OracleDatabase/dockerfiles/buildDockerImage.sh

```diff
@@ -103,7 +111,7 @@ elif [ $EXPRESS -eq 1 ] && [ "$VERSION" != "11.2.0.2" ]; then
   exit 1;
 else
   EDITION="xe";
-  DOCKEROPS="--shm-size=1G";
+  DOCKEROPS="--shm-size=1G $DOCKEROPS";
```

Comment:
Can you double-check the logic here? I have this feeling that you're only adding the `$DOCKEROPS` parameter if you're building the XE image.